Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -187,6 +192,8 @@ extension Target.Error: FixableError {
             return "rename the directory '\(path)'\(problem.fix ?? "")"
         case .mixedSources(_):
             return "use only a single language within a target"
+        case .duplicateTargets(let targets):
+            return "remove the duplicate target definitions in Package.swift: " + targets.joined(separator: ", ")
```

Comment:
Just return nil from here, I plan to remove "fix" and make main errors more descriptive. These are not really helpful.